### ISSUE TITLE
Bump version to Yae Ari Update (2.4.8)

### ISSUE THREAD

**Issue by Bronya-Rand** [imgradeone/DDLCModTemplate-Chinese-next#2]
To maintain that this template is similar to the one from the main DDLCModTemplate2.0 repository, this issue is mainly here to address you to update the next template over to 2.4.8 or the Yae Ari Update that has been released a bit ago. 

This update mainly addresses a few things like
- Setting config.developer to `"auto"` than False
- Updates the lockdown code in splash.rpy and the lockdown RPY itself
- Pronoun support for players who go by She/Her, They/Them
to name a few.

This also makes sure the template is locked out for any new Ren'Py released after 7.4.6 as this bug is still present in the engine (partially in 7.4.7 and above), and for quality assurance purposes is disabled for lower Ren'Py versions that do work.

**Comment by Bronya-Rand** [comment]
Also I noticed that `guide.pdf` is missing from the repository. IDK if this is being re-translated into Chinese but it is a bit vital to have said PDF in the repo since the Readme still cites it as the way to make a mod available on Android. You can fetch that guide either from 2.4.6 (Sucrose of Time Update) or during the migration to 2.4.8 (Yae Ari Update) unless a doc is needed.

**Comment by imgradeone** [comment]
For me, I would like to migrate to 2.4.8 and add Android Mod Document to our website (https://docs.dokimod.cn) soon.

**Comment by Bronya-Rand** [comment]
I see. If that's the case I recommend updating the readme when that's done to it. Also just to add things to the docs in question

1st Doc page (开始准备环境)
2.3.1 is still maintained under 2.3.1-uX updates (recent being 2.3.1-u7 [Miko Ana Update]). Before I thought this will be left as-is but with people asking for a non-android template still, I kept making the releases from 2.4.X avail on 2.3.1 as 2.3.1-uX updates with no Android code within it. 

`scripts.rpa` can be copied to the game directory nor does it need to be removed on a clean copy of the game with my template. That has been fixed long ago.

2nd Doc page (第一个简单的脚本)

All is good though I recommend updating the doc sample code for script.rpy to the latest since a comment is addressed to what it actually is used for.

Apart from that, all is good from me.
